recipe_example_project.mdx

```diff
@@ -0,0 +1,269 @@
+</video>
+
+Once our prompt template has been created, we can retrieve it from PromptLayer.
+
+```python
+recipe_template = promptlayer.prompts.get("recipe_template")
+system_content = recipe_template['messages'][0]['prompt']['template']
+user_content_template = recipe_template['messages'][1]['prompt']['template']
+```
+
+To use our template, we'll create a `variables` dictionary containing the values we want to pass in. Using these values, we'll construct the full prompt then pass it to OpenAI to obtain a recipe suggestion.
+
+```python
+variables = {
+    'food': food,
+    'recipes_string': '\n\n'.join(recipes_list)
+}
+
+response, pl_request_id = openai.ChatCompletion.create(
+    model='gpt-3.5-turbo',
+    messages=[
+        {
+            'role': 'system',
+            'content': system_content
+        },
+        {
+            'role': 'user',
+            'content': user_content_template.format(**variables)
+        },
+    ],
+    temperature=0.5,
+    max_tokens=1024,
+    return_pl_id=True
+)
+print(response.choices[0].message.content)
+```
+
+Lastly, we'll associate the generated response with this specific prompt template in PromptLayer for tracking.
+
+```python
+promptlayer.track.prompt(
+    request_id=pl_request_id,
+    prompt_name='recipe_template',
+    prompt_input_variables=variables
+)
+```
+
+After running this script you'll be able to see the final recipe suggestion in PromptLayer!
+
+![Recipe Request Log Screenshot](images/recipe-request-log.png)
+
+# Improving Results with Prompt Engineering
+
+Things are working, but let's tweak our prompt a bit to see if we can get some better results. Instead of `I like {food}`, try using the text `My favorite food is {food}`.
+
+This simple change in the phrasing of the prompt can significantly alter the response from the AI. Using more affirmative language like "My favorite food is" rather than just "I like" puts more emphasis on the user's preference. This signals to the AI that it should pay close attention to tailoring the suggestion to that specific food.
+
+**Prompt engineering** involves systematically experimenting with different ways of phrasing the input prompt to find what works best. Some key aspects to tweak can include:
+- Using more natural conversational language
+- Providing additional context to guide the AI
+- Emphasizing key information through wording
+- Adjusting the tone, perspective, and personality
+
+PromptLayer provides useful tools for prompt engineering. By centralizing prompt templates in the Prompt Registry, it makes it easy to track different versions of a prompt. And the analytics provide visibility into how well different prompts are performing.
+
+This iterative process of refining prompts is crucial for getting the most out of AI systems. Even small tweaks can sometimes lead to significant improvements. Prompt engineering combines art and science to unlock the full potential of language models like GPT-3.
+
```

Comment:
Can we add two sub-sections here:

## Prompt Template Metadata

[adding model metadata to the template and code to pull it down]

## Prompt Template Labels

Using a "prod" label so it's easier to experiment